### ISSUE TITLE
class name formet hyphenatedbem not working

### ISSUE THREAD

**Issue by PXLRic** [sasstools/sass-lint#730]
What version of Sass Lint are you using?
1.7.0

Please include any relevant parts of your configuration
  class-name-format: hyphenatedbem

What did you do? Please include the actual source code causing the issue.
hyphenatedbem does not allow these names (or the samples ones here: https://github.com/sasstools/sass-lint/blob/master/docs/rules/class-name-format.md#example-6)

.input__radio-group
.input__label

.input {
	align-items: baseline;
        ....

	&__radio-group {
		display: flex;
                ....
	}

        &__label {
              margin-bottom: 10px;
              ...
        }
}

What actually happened? Please include any error messages given to you by Sass Lint.
.input__label should be written in lower case with hyphens

If you're using a IDE plugin have you tried the CLI too?
PHPStorm Sass Lint plugin

**Comment by DanPurdy** [comment]
@PXLRic I don't think you have your configuration set correctly.

Can you copy and paste exactly what you have.

The warning you are seeing is the warning you would get when using `hyphenatedlowercase`

If you want your config to work for `hyphenatedbem` it needs to look like the following:

```yaml
class-name-format:
    - 1
    -
      convention: hyphenatedbem
```

Could you clarify if this works for you? Thanks! 👍

**Comment by PXLRic** [comment]
That's fixed it, thanks. Not too familiar with yml

**Comment by DanPurdy** [comment]
No problem 👍
